vendor/re/src/re.mli

```diff
@@ -167,8 +167,8 @@ val split_gen :
   re -> string -> string gen
 
 type split_token =
-  [ `Text of string  (** Text between delimiters *)
-  | `Delim of Group.t (** Delimiter *)
+  [ `Text of string  (* Text between delimiters *)
+  | `Delim of Group.t (* Delimiter *)
```

Comment:
Is it possible to disable a warning to avoid doing this (via `(flags (:standard -w -N))`)? Since this code is vendored, I'd rather modify as little as possible, otherwise the modifications will be lost when we import a new version.